__tests__/app.test.js

```diff
@@ -103,6 +104,58 @@ describe("API Testing", () => {
 					});
 			});
 		});
+		describe("Patch article", () => {
+			test("Patch: 200, Return updated article", () => {
+				const newVote = -15;
+				return request(app)
+					.patch("/api/articles/1")
+					.send({ inc_votes: newVote })
+					.expect(200)
+					.then(({ body }) => {
+						expect(body.article_id).toBe(1);
+						expect(body.votes).toBe(85);
+						expect(body).toHaveProperty("author");
+						expect(body).toHaveProperty("title");
+						expect(body).toHaveProperty("article_id");
+						expect(body).toHaveProperty("body");
+						expect(body).toHaveProperty("topic");
+						expect(body).toHaveProperty("created_at");
+						expect(body).toHaveProperty("votes");
+						expect(body).toHaveProperty("article_img_url");
+					});
+			});
+			test("Err: 404, Returns error 404 when given a valid but non existant article id", () => {
+				const newVote = -15;
+				return request(app)
+					.patch("/api/articles/9999999")
+					.send({ inc_votes: newVote })
+					.expect(404)
+					.then((response) => {
+						expect(response.body.msg).toBe("article does not exist");
+					});
+			});
+			test("Err: 400, should be able to handle a get request when there are no matches with invalid url", () => {
+				const newVote = "string";
+				return request(app)
+					.patch("/api/articles/article")
+					.send({ inc_votes: newVote })
+					.expect(400)
+					.then((response) => {
+						expect(response.body.msg).toBe("Bad request");
+					});
+			});
+			test("Err: 400, Return error 400 if not provided with a inc_votes key", () => {
+				return request(app)
+					.patch("/api/articles/2/")
+					.send({
+						body: "hello world",
+					})
+					.expect(400)
+					.then((response) => {
+						expect(response.body.msg).toBe("Bad request");
+					});
+			});
```

Comment:
what happens if inc_votes' value is: "two" - it would probably 400 right? Have a test for it :)